boilerplate/app/models/Episode.ts

```diff
@@ -31,9 +31,10 @@ export const EpisodeModel = types
   .views((episode) => ({
     get datePublished() {
       try {
-        return formatDate(episode.pubDate)
+        const formatted = formatDate(episode.pubDate)
+        return { textLabel: formatted, accessibilityLabel: `Published on ${formatted}` }
```

Comment:
nit: can remove "on", (e.g., "Published 2/12/2022")

Reply:
Should this string template be in i18n strings?

Reply:
Ah good catch